src/Image/__tests__/Image.test.js

```diff
@@ -0,0 +1,193 @@
+
+  const { getByAltText } = render(<Image image={imageWithPreferred} />)
+
+  const img = getByAltText('Serena Brook opens our show at The Town Hall')
+
+  expect(img).toBeInTheDocument()
+  expect(img).toHaveAttribute(
+    'srcSet',
+    expect.stringContaining(expected.srcSet)
+  )
+  expect(img).toHaveAttribute('src', expect.stringContaining(expected.src))
+})
+
+test('prioritizes the aspectRatio prop over the preferredAspectRatio in the data', () => {
+  const expected = objectMother()
+  expected.srcSet =
+    'https://img.apmcdn.org/c2c452354fbff94d720ba8f86e2c71ba7427b306/square/5ecd52-20181220-serena-brook-opens-our-show-at-the-town-hall.jpg 400w,https://img.apmcdn.org/c2c452354fbff94d720ba8f86e2c71ba7427b306/square/de193e-20181220-serena-brook-opens-our-show-at-the-town-hall.jpg 600w,https://img.apmcdn.org/c2c452354fbff94d720ba8f86e2c71ba7427b306/square/7cb7e2-20181220-serena-brook-opens-our-show-at-the-town-hall.jpg 1000w,https://img.apmcdn.org/c2c452354fbff94d720ba8f86e2c71ba7427b306/square/822d4e-20181220-serena-brook-opens-our-show-at-the-town-hall.jpg 1400w,https://img.apmcdn.org/c2c452354fbff94d720ba8f86e2c71ba7427b306/square/f977a8-20181220-serena-brook-opens-our-show-at-the-town-hall.jpg 2000w'
+
+  const { getByAltText } = render(
+    <Image image={imageWithPreferred} aspectRatio="square" />
+  )
+
+  const img = getByAltText('Serena Brook opens our show at The Town Hall')
+
+  expect(img).toBeInTheDocument()
+  expect(img).toHaveAttribute(
+    'srcSet',
+    expect.stringContaining(expected.srcSet)
+  )
+  expect(img).toHaveAttribute('src', expect.stringContaining(expected.src))
+})
+
+test('takes in a "sizes" string to specify image behavior based on viewport', () => {
+  const expected = objectMother()
+
+  const { getByAltText } = render(
+    <Image image={image} sizes="(min-width: 960px) 720px, 100vw" />
+  )
+
+  const img = getByAltText('Serena Brook opens our show at The Town Hall')
+
+  expect(img).toBeInTheDocument()
+  expect(img).toHaveAttribute('src', expect.stringContaining(expected.src))
+  expect(img).toHaveAttribute('sizes', expect.stringContaining(expected.sizes))
+})
+
+test('allows you to set the class with the elementClass property', () => {
+  const expected = objectMother()
+
+  const { getByAltText } = render(
+    <Image
+      image={imageWithPreferred}
+      elementClass="test"
```

Comment:
This is the same className that is in the objectMother  should probably be different